File: 백준/Bronze/28281. 선물/선물.js
const fs = require('fs');
const filePath = process.platform === 'linux' ? '/dev/stdin' : 'run/input.txt';
const input = fs.readFileSync(filePath).toString().trim().split('\n');

const [n, x] = input[0].split(' ').map(Number);
const a = input[1].split(' ').map(Number);

let r = Infinity;

for (let i = 1; i < n; i++) {
  const c = a[i - 1] * x + a[i] * x;
  if (r > c) {
    r = c;
  }
}

console.log(r);

File: 백준/Bronze/28281. 선물/선물.py
n, x = map(int, input().split())
a = list(map(int, input().split()))

r = float('inf')

for i in range(1, n):
    c = a[i - 1] * x + a[i] * x
    if r > c:
        r = c

print(r)